docs/src/index.md

```diff
@@ -15,6 +15,23 @@ It is written in [Julia](https://julialang.org/) and depends on
 [Nemo](https://nemocas.github.io/Nemo.jl/stable/) and therefore on
 [AbstractAlgebra](https://nemocas.github.io/AbstractAlgebra.jl/stable/).
 
+
+For the mathematical background, consider that there are many interesting
+*families* of finite groups. For example, the matrix groups
+$\mathrm{GL}_n(\mathbb{F}_q)$ or $\mathrm{SL}_n(\mathbb{F}_q)$, for $n>1$ and
+$q$ a prime power. These groups have many properties in common. It turns out
+that for a fixed rank (say $n=2$) it is possible to parametrize the conjugacy
+classes and irreducible characters of these group in terms of $q$, and to
+write this down into a so-called *generic character table*. This was first
+done by Schur for $\mathrm{SL}_2(\mathbb{F}_q)$.
```

Comment:
I think in the future "Schur" should also be a reference here.

Reply:
Yeah but I'll happily leave adding all the relevant bibitems and then linking to them to you, in a future follow-up PR ;-)